PseudoLocalizer.Core.Tests/POProcessorTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+namespace PseudoLocalizer.Core.Tests
+{
+    [TestFixture]
+    public class POProcessorTests
+    {
+        private const string Test1FileName = "Test1.po";
+        private const string Test2FileName = "Test2.po";
+        private const string OutputFileName = "out.po";
+        
+        [SetUp]
+        public void SetUp()
+        {
+            DeleteOutputFile();
+        }
+
+        [Test]
+        public void ShouldLeaveTheFileUntouchedWhenUsingAnIdentityTransformation()
+        {
+            using (var inputStream = new FileStream(Test1FileName, FileMode.Open, FileAccess.Read))
+            using (var outputStream = new FileStream(OutputFileName, FileMode.Create, FileAccess.Write))
+            {
+                var processor = new POProcessor("en");
+                processor.Transform(inputStream, outputStream);
+            }
```

Comment:
This changes the behaviour and causes problems with the unit test, because the stream resource cleanup doesn't occur at the right time:
`System.IO.IOException : The process cannot access the file 'C:\path\to\Pseudolocalizer\PseudoLocalizer.Core.Tests\bin\Debug\netcoreapp3.0\out.po' because it is being used by another process.`